recipe/meta.yaml

```diff
@@ -14,13 +14,14 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<37]
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
   host:
     - python
     - pip
```

Comment:
`setuptools` is missing, see https://github.com/flask-admin/flask-admin/blob/3f9bda85b3a6040ee0327474fbf5e3515467b58a/setup.py#L6